### PR TITLE
[RI2 Ch3] Update purpose of RI2 wrt. RC2

### LINKED ISSUE #2125: [RI2 Ch3] Update purpose of RI2 wrt. RC2
RI2 currently states that RC2 is developed and tested using RI2, which is incorrect.

RI2, using Kuberef, should be verified by RC2, while also providing feedback to RC2 based on the results.
Details on the feedback process is still TBD

### PR DESCRIPTION
**Pull request by michaelspedersen**
Closes #2125  
Changes wording from RI2 feeding into RC2, to RC2 verifying RI2

### CODE REVIEW

**Review by tomkivlin — APPROVED** [review]

**Review by gkunz — APPROVED** [review]

**Comment by rihabbanday** [comment]
Need to check if we can exclude OVP section

File: doc/ref_impl/cntt-ri2/chapters/chapter03.md [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 The purpose of this chapter is to list and describe the requirements on labs used to deploy and run the RI2. Overall, the RI2 serves two purposes which guide the definition of requirements in this chapter:
 
-1. RI2 serves as a baseline and proving ground for developing the [Reference Conformance 2 (RC2) specification](../../../ref_cert/RC2) as well as the [Cloud Native OVP](https://wiki.lfnetworking.org/x/agCDAQ) program.
+1. RI2 serves as a baseline and proving ground for the [Reference Architecture 2 (RA2) specification](../../../ref_arch/kubernetes). It is verified through the [Reference Conformance 2 (RC2) specification](../../../ref_cert/RC2).
```
Comment (pgoyal01) [inline doc/ref_impl/cntt-ri2/chapters/chapter03.md:19]:
```suggestion
1. RI2 is an implementation of and a proving ground for the [Reference Architecture 2 (RA2) specification](../../../ref_arch/kubernetes). RI2 is verified against RA2 through the [Reference Conformance 2 (RC2) specification](../../../ref_cert/RC2).
```

**Comment by pgoyal01** [comment]
As per discussion in the RI2 meeting of 2021-01-13, add:

1. RI2 will align itself with the implementation work conducted within the Cloud Native OVP (reference?) program.

**Comment by michaelspedersen** [comment]
> As per discussion in the RI2 meeting of 2021-01-13, add:
> 
> 1. RI2 will align itself with the implementation work conducted within the Cloud Native OVP (reference?) program.

Added this to the chapter, alongside a few minor changes (nit)

File: doc/ref_impl/cntt-ri2/chapters/chapter03.md [hunk 2]
```diff
@@ -14,12 +14,13 @@
 <a name="3.1"></a>
 ## 3.1 Introduction
 
-The purpose of this chapter is to list and describe the requirements on labs used to deploy and run the RI2. Overall, the RI2 serves two purposes which guide the definition of requirements in this chapter:
+The purpose of this chapter is to list and describe the requirements on labs used to deploy and run the RI2. Overall, the RI2 serves the following purposes which guide the definition of requirements in this chapter:
```
Comment (pgoyal01) [inline doc/ref_impl/cntt-ri2/chapters/chapter03.md:17]: nit
```suggestion
The purpose of this chapter is to list and describe the requirements on labs used to deploy and run the RI2. Overall, the RI2 serves the following purpose which guide the definition of requirements in this chapter:
```

File: doc/ref_impl/cntt-ri2/chapters/chapter03.md [hunk 3]
```diff
@@ -14,12 +14,13 @@
...
 
-1. RI2 serves as a baseline and proving ground for developing the [Reference Conformance 2 (RC2) specification](../../../ref_cert/RC2) as well as the [Cloud Native OVP](https://wiki.lfnetworking.org/x/agCDAQ) program.
+1. RI2 is an implementation of and a proving ground for the [Reference Architecture 2 (RA2) specification](../../../ref_arch/kubernetes). RI2 is verified against RA2 through the [Reference Conformance 2 (RC2) specification](../../../ref_cert/RC2).
+1. RI2 will align itself with the implementation work conducted within the [Cloud Native OVP](https://wiki.lfnetworking.org/x/agCDAQ) program.
```
Comment (tomkivlin) [inline doc/ref_impl/cntt-ri2/chapters/chapter03.md:20]: OVP program expanded would be OPNFV verification program program?
```suggestion
1. RI2 will align itself with the implementation work conducted within the [Cloud Native OVP](https://wiki.lfnetworking.org/x/agCDAQ).
```

File: doc/ref_impl/cntt-ri2/chapters/chapter03.md [hunk 4]
```diff
@@ -14,12 +14,13 @@
...
 1. RI2 enables software vendors of CNFs and cloud container platforms to deploy a reference implementation in their development environment to develop and validate against.
```
Comment (pgoyal01) [inline doc/ref_impl/cntt-ri2/chapters/chapter03.md:21]: nit - why do we care which environment the vendors deploy RI on
```suggestion
1. RI2 enables software vendors of CNFs and cloud container platforms to deploy a reference implementation in their environment to develop and validate against.
```

**Review by tomkivlin — CHANGES_REQUESTED** [review]

File: doc/ref_impl/cntt-ri2/chapters/chapter03.md [hunk 5]
```diff
@@ -14,12 +14,13 @@
...
 
-As a result of those two main use cases, RI2 must be deployable in different lab environments:
+As a result of these, RI2 must be deployable in different lab environments:
```
Comment (pgoyal01) [inline doc/ref_impl/cntt-ri2/chapters/chapter03.md:23]:
```suggestion
As a result, RI2 must be deployable in different environments:
```

File: doc/ref_impl/cntt-ri2/chapters/chapter03.md (hunk 2 above)
Comment (michaelspedersen) [reply doc/ref_impl/cntt-ri2/chapters/chapter03.md:17]: Shouldn't it be kept plural as there are three bullets below it?

File: doc/ref_impl/cntt-ri2/chapters/chapter03.md [hunk 6]
```diff
@@ -14,12 +14,13 @@
...
 
 1. internal development labs of software vendors, operators and 3rd party test labs,
 1. public community labs (e.g. OPNFV),
```
Comment (pgoyal01) [inline doc/ref_impl/cntt-ri2/chapters/chapter03.md:26]: if not deleted -- a precious suggestion is rejected, then correct typo
```suggestion
1. public community labs (e.g. Anuket (formerly OPNFV))
```

Comment (michaelspedersen) [reply doc/ref_impl/cntt-ri2/chapters/chapter03.md]: @pgoyal01 I took a bit from both of the proposals. Let me know if that works

File: doc/ref_impl/cntt-ri2/chapters/chapter03.md (hunk 5 above)
Comment (gkunz) [reply doc/ref_impl/cntt-ri2/chapters/chapter03.md:23]: fine with me.

File: doc/ref_impl/cntt-ri2/chapters/chapter03.md (hunk 2 above)
Comment (gkunz) [reply doc/ref_impl/cntt-ri2/chapters/chapter03.md:17]: I have to admit that both, singular and plural, sound a bit weird to me - but I am not a native speaker. Just as an alternative proposal, how about something like "... Overall, the RI2 has the following objectives which guide..."?

**Review by pgoyal01 — APPROVED** [review]
approved

File: doc/ref_impl/cntt-ri2/chapters/chapter03.md (hunk 2 above)
Comment (pgoyal01) [reply doc/ref_impl/cntt-ri2/chapters/chapter03.md:17]: @michaelspedersen yes -- sorry my mistake.

**Review by tomkivlin — APPROVED** [review]

**Pull request merged**